tests/trx2junit.Tests/Internal/TrxParserTests/Parse.cs

```diff
@@ -21,6 +21,14 @@ public void Parse_MsTest___OK()
 
             Assert.IsNotNull(actual);
         }
+
+        [Test]
+        public void Parse_MsTestWithWarnings___OK()
```

Comment:
Nit: Can you please name the test `Parse_MsTest_with_warnings___OK`?
So it follows the naming-scheme of the test. But I just realized that the naming-scheme is not documented in this repo -- I'll fix this.

Reply:
will do